src/components/ansatte/Refusjon.tsx

```diff
@@ -7,29 +7,84 @@ import { HjelpetekstRefusjon } from "./HjelpetekstRefusjon";
 import { validateMaksBeloep } from '../validering/validateMaksBeloep';
 
 export const Refusjon = (props: AnsattID) => {
-  const {ansatte, setAnsatte} = useAppStore();
+  const { ansatte, setAnsatte } = useAppStore();
   const a = ansatte.find(a => a.id === props.id);
-  const handleChange = (evt) => {
+  const [value, updateValue] = useState('');
+  var currentVal = 0;
+
+  const getSpecialCharsOnSides = (x: string, cursorPosition: number) => {
+    const specialCharsLeft = x.substring(0, cursorPosition).replace(/[0-9]/g, '').length;
+    const specialCharsRight = x.substring(cursorPosition).replace(/[0-9]/g, '').length;
+    return [specialCharsLeft, specialCharsRight]
+  }
+
+  const cleanNumber = (strNumber: string): string => {
+    const val = strNumber.replace(/,/g, ".");
+    return val.replace(/\s/g, "");
+  }
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Dersom du flytter valideringsalgoritmen ut i en egen metode kan du lettere lage test som bare tester på hva brukene skrev inn. feks:
test "1" gir "1"
test "12" gir "12"
test "123" gir "123"
test "1234" gir "123 4"
test "12345" gir "123 45"

etc

Reply:
Selve formateringen er disse linjene:
```
var formatter = new Intl.NumberFormat("nb-NO", { minimumFractionDigits: 2 });
const strCurrentVal = formatter.format(x);
```
resten er plassering av cursoren.

Reply:
Er det sjekk på at brukeren har tastet inn alle "typiske" verdier?

Reply:
Jepp! Den fungerer som for enkeltinnsending. Om man ikke skriver inn et siffer så endres ikke verdien. Og ja, det er enhetstest for det :-)